Core/uWebshop.Domain/Model/OrderDTO/Order.cs

```diff
@@ -44,7 +44,7 @@ public class Order : IConvertibleToOrderInfo
 	    public DateTime? FulfillDate;
 		public PaymentInfo PaymentInfo;
 		public int PaymentProviderPrice;
-		public int? PaymentProviderOrderPercentage;
+		public decimal? PaymentProviderOrderPercentage;
```

Comment:
I don't care how to store/process the percentage, as long as the field is clear for end-users.